data/device-tests/caldigit-ts4.json

```diff
@@ -4,6 +4,7 @@
   "steps": [
     {
       "url": "https://fwupd.org/downloads/dbeeba12332296ae68eaa104ed3b4b437659bfe027e347010a87c75bcea87292-CalDigit-TS4-combind-050322-secure-Cus-22.cab",
+      "emulation-url": "https://fwupd.org/downloads/e755191ff3c7d26777273262eff0be266758ebe5b79685cd44d14d491ab6107a-CalDigit-TS4-combind-050322-secure-Cus-22.zip",
```

Comment:
I wonder if the device-tests json is better stored in https://github.com/fwupd/fwupd-test-firmware

Reply:
I wondered that too; but it's kinda huge in some cases -- and it's the kind of thing that needs to be updatable/removable/embargoed by the vendor in the same way as firmware.

Reply:
Okay makes sense.